src/utils/utils.js

```diff
@@ -555,6 +557,15 @@ export function containsUnbreakable(node) {
     }
     return isUnbreakable(node) || containsUnbreakable(node.firstChild);
 }
+export function isFontAwesome(node) {
+    const faClasses = className => ['fa', 'fab', 'fad', 'far'].includes(className);
+    return (
+        node &&
+        (node.nodeName === 'I' || node.nodeName === 'SPAN') &&
+        node.attributes.class &&
+        node.attributes.class.textContent.split(/(\s)/).some(faClasses)
```

Comment:
You can use `classList` for this:
```js
[...node.classList].some(faClasses)
```
In the spirit of minimal LOC, this whole function can actually be written like so:
```js
return [...(node?.classList || [])].some(['fa', 'fab', 'fad', 'far'].includes) && ['I', SPAN'].includes(node.nodeName);
```
:eyes:

Reply:
:eyes: